### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by andylushman**
Added team page by creating:
* `team.html.slim`
* `team_member_card`
* `team_member_list`
* `static data with team_members.yml`

### CODE REVIEW

File: app/data/team_members.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+members:
```
Comment (scpaye506) [inline app/data/team_members.yml]: we don't want to use static data for this. we want to get the team members the same way the old team page gets them.

**Comment by scpaye506** [comment]
I think we need to add a redirect from the old team path to the new one

**Comment by andylushman** [comment]
Just added a redirect: `get "/get-involved/team", to: redirect("/about/team")`

File: app/data/team_members.yml (hunk 1 above)
Comment (andylushman) [reply app/data/team_members.yml]: I think I have it done correctly now but it's taking me some time to figure out how to add the seed data.

Comment (andylushman) [reply app/data/team_members.yml]: I think I have it done correctly now but it's taking me some time to figure out how to add the seed data. I added a `User` to `seeds.rb` and ran `rake db:seed` but the database doesn't get populated.

File: app/views/components/_team_member_card.html.slim [hunk 2]
```diff
@@ -0,0 +1,5 @@
+= render 'components/card' do
+  .TeamMemberCard
+    .TeamMemberCard-image(style="background-image: url(#{user.avatar.url[:thumb]})")
```
Comment (scpaye506) [inline app/views/components/_team_member_card.html.slim]: this should be `user.avatar.url(:thumb)`, otherwise looks good

Comment (andylushman) [reply app/views/components/_team_member_card.html.slim]: 👍

**Pull request merged**